terraform/aws/development-infrastructure/nbs-legacy/Docker/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# This Dockerfile is used to build NBS 6.0 Image locally and pushed to a ECR Repository
+
+# Use Windows Server Core as the base image
+FROM mcr.microsoft.com/windows/servercore:ltsc2019
+
+# Run Shell as Powershell
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+# Set environment variables and AWS credentials and region
+ENV JAVA_HOME="D:\wildfly-10.0.0.Final\Java\jdk1.8.0_181" \
+    JBOSS_HOME="D:\wildfly-10.0.0.Final"
+
+# Set symbolic link as D drive using HKEY_LOCAL_MACHINE (HKLM)
+RUN New-Item -ItemType directory -Path C:\nbs ; \
+    New-ItemProperty -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\DOS Devices' -Name 'D:'  -Value '\??\C:\nbs' -PropertyType String
+
+# Configure NBS from Local Directory
+# NOTE: 
+# 1) Verify Zip File is downloaded from S3 and place in same directory, example: wildfly-10.0.0.Final-6.0.15.1.zip
+# 2) Delete wildfly*/bin/hs_err_pid2604.mdmp and/or any file name containing "err"
+WORKDIR /nbs
+COPY wildfly-10.0.0.Final-6.0.15.1.zip wildfly-10.0.0.Final-6.0.15.1.zip
+RUN Expand-Archive -Path "wildfly-10.0.0.Final-6.0.15.1.zip" ; \
```

Comment:
This should be a dynamic variable based on the version of classic NBS which may change in the future